Sources/ForageSDK/ForageSDK.swift

```diff
@@ -41,8 +41,23 @@ public class ForageSDK {
         LDManager.shared.initialize(self.environment)
         // TODO: Maybe move this shared logger call!
         VGSCollectLogger.shared.disableAllLoggers()
+       
         self.service = LiveForageService()
+        let isUnitTesting = ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil
+
+        if !isUnitTesting {
+        SentrySDK.start { options in
+            options.dsn = "https://8fcdd8dc94aa892ed8fd4cdb20db90ee@o921422.ingest.sentry.io/4505665631813632"
+            options.debug = false
+            options.environment = String(describing: self.environment)
+            let httpStatusCodeRange = HttpStatusCodeRange(min: 400, max: 599)
+            options.failedRequestStatusCodes = [ httpStatusCodeRange ]
+            options.tracesSampleRate = 1.0
+            options.enableTracing = true
+        }
+        }
     }
```

Comment:
nit: can we re-indent - there's a shortcut in xcode, i believe `Select text > CTRL + I`

we'll add auto formatting/linting soon! https://linear.app/joinforage/issue/FX-381/add-linting-formatting-scripts-for-forage-ios-sdk

Reply:
67cc470727416a02fb420a37df7d21b1e48dea4b